jenkins/scripts/integration_test_clean.sh

```diff
@@ -51,11 +52,13 @@ if [[ "${OS_REGION_NAME}" == "Fra1" ]] || [[ "${OS_PROJECT_NAME}" == "dev2" ]];
     TEST_EXECUTER_IP="${FLOATING_IP}"
 fi
 
+if [[ -n "${SSH_JUMP_HOST}" ]]; then
```

Comment:
In this if, you should only be adding the proxy line to `ssh_options` and not duplicate the actual commands.